bundles/core/org.eclipse.smarthome.core.persistence/src/main/java/org/eclipse/smarthome/core/persistence/internal/PersistenceManagerImpl.java

```diff
@@ -0,0 +1,420 @@
+                                    logger.debug("Restored item state from '{}' for item '{}' -> '{}'",
+                                            new Object[] {
+                                                    DateFormat.getDateTimeInstance()
+                                                            .format(historicItem.getTimestamp()),
+                                                    item.getName(), historicItem.getState().toString() });
+                                    return;
+                                }
+                            } else if (service != null) {
+                                logger.warn(
+                                        "Failed to restore item states as persistence service '{}' can not be queried.",
+                                        serviceName);
+                            }
+                        }
+                    }
+                }
+            }
+        }
+    }
+
+    /**
+     * Creates and schedules a new quartz-job.
+     *
+     * @param modelName the name of the model
+     * @param strategies a collection of strategies
+     */
+    private void createTimers(final String modelName, List<SimpleStrategy> strategies) {
+        for (SimpleStrategy strategy : strategies) {
+            if (strategy instanceof SimpleCronStrategy) {
+                SimpleCronStrategy cronStrategy = (SimpleCronStrategy) strategy;
+                String cronExpression = cronStrategy.getCronExpression();
+                final CronExpression expression;
+                try {
+                    expression = new CronExpression(cronExpression);
+                } catch (final ParseException ex) {
+                    logger.warn("Cannot parse cron expression ({}).", cronExpression, ex);
+                    continue;
+                }
+
+                final PersistItemsJob job = new PersistItemsJob(this, modelName, cronStrategy.getName());
+                if (persistenceJobs.containsKey(modelName)) {
+                    persistenceJobs.get(modelName).add(job);
+                } else {
+                    final Set<Runnable> jobs = new HashSet<>();
+                    jobs.add(job);
+                    persistenceJobs.put(modelName, jobs);
+                }
+
+                scheduler.schedule(job, expression);
+                logger.debug("Scheduled strategy {} with cron expression {}", cronStrategy.getName(), cronExpression);
+            }
+        }
+    }
+
+    /**
+     * Delete all {@link Job}s of the group <code>persistModelName</code>
+     *
+     * @throws SchedulerException if there is an internal Scheduler error.
+     */
+    private void removeTimers(String persistModelName) {
+        if (!persistenceJobs.containsKey(persistModelName)) {
+            return;
+        }
+        for (final Runnable job : persistenceJobs.get(persistModelName)) {
+            boolean success = scheduler.remove(job);
```

Comment:
I just started testing this PR, but in this line success is always false and I get:
```
2016-11-02 15:25:55.180 [WARN ] [o.e.s.c.p.PersistenceManager  :317  ] - Failed to delete cron job for dbId 'rrd4j```
in my log (when updating the file rrd4j.persist).
This is probably an issue in the ESH scheduler implementation, but I'd like to know if this has worked for you? In my tests, the scheduler keeps creating new jobs and never removes them again...